packages/strapi/bin/strapi.js

```diff
@@ -79,7 +79,12 @@ program
 
 // `$ strapi new`
 program
-  .command('new [name]')
+  .command('new <directory>')
+  .option('--no-run', 'Disable quickstart app start after creation')
```

Comment:
I think we have to update this message.
I don't really understoud the first time and I had to check the code to understand (in fact I know, but I was not sure).
`Disable the application start after the quickstart app creation`
Something like that is more understandable for me.
> This is a proposal, let discuss about it.